src/features/nrfutil91.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import { getModule } from '@nordicsemiconductor/pc-nrfconnect-shared/nrfutil';
+
+interface IMEINumber {
+    devices: {
+        imei_numbers: string[];
+        serial_number: string;
+    }[];
+}
+
+const modemFirmware = '/home/papi/temp.zip';
+
+const writeIMEI = async (serialNumber: string, imei: string) => {
+    const box = await getModule('91');
+    const args: string[] = [
+        '--serial-number',
+        serialNumber,
+        '--modem-firmware',
+        modemFirmware,
```

Comment:
this should be a parameter not hard coded as use might pass in the PTI or the Normal Modem FW